esphomeRemote/esphomeRemoteLight.h

```diff
@@ -2,30 +2,137 @@
 #include "DisplayUpdateInterface.h"
 #include "MenuTitle.h"
 #include "FriendlyNameEntity.h"
+#include <sstream>
 
 #pragma once
 
+/* Home Assistant entities can have these modes in color_mode/supported_color_mode.
+ * Depending on the mode the light supports we want to show sliders or not.
+ *
+ * |color_mode | supports brightness | supports color_temp |
+ * |---------- | ------------------- | ------------------- |
+ * | UNKNOWN   |          -          |          -          |
+ * | ONOFF     |          -          |          -          |
+ * | BRIGHTNESS|          x          |          -          |
+ * | COLOR_TEMP|          x          |          x          |
+ * | HS        |          x          |          -          |
+ * | RGB       |          x          |          -          |
+ * | RGBW      |          x          |          -          |
+ * | RGBWW     |          x          |          -          |
+ * | WHITE     |          x          |          -          |
+ * | XY        |          x          |          -          |
+ */
+

```

Comment:
can we make an enum for light features that matches these? For example
```
enum LightFeature {
  case brightness
  case temperature
  case rgb
}
```
and then each light can return an array of which features it has which maps to a MenuItem

Reply:
I like the enum idea but I don't think we need an array:
- if a lights supports rgb it also supports brightness and temperature
- if a light supports temperature it supports brightness
- if a lights supports birghtnes only then there is no rgb/temperature support
- if a light has color_mode "brightness" it only supports brightness adjustments
- if a light has color_mode "onoff" or "unknown" it doesn't support any adjustments

So I think it's rgb > temperature > brightness > onoff and every light would have one of these values:

```  
enum LightFeature {  
    onoff,
    brightness,  
    temperature,  
    rgb  
}  
```
Or is it more complicated? I don't have a color light right now so unfortunately I cannot try this...

Reply:
there's RGB and RGBWW bulbs. rgbww have a separate white channel that can do more accurate color temp matching

Reply:
oops I didn't know that. So that means not all RGB lights support temperature. Thx for clearing that up!